File: src/app/model/schedule.ts
import {Activity} from './activity';

export class Schedule {

  id: number;
  planningDate: Date;
  relationId: number;
  isFree: boolean;
  activities: Activity[];
  holiday: boolean;

  constructor(id: number, planningDate: Date, relationId: number, activities: Activity[], holiday: boolean) {
    this.id = id;
    this.planningDate = planningDate;
    this.relationId = relationId;
    this.activities = activities;
    this.holiday = holiday;
  }
}

File: schedule/src/main/java/com/monster/schedule/model/Schedule.java
package com.monster.schedule.model;

import java.io.Serializable;
import java.time.LocalDate;
import java.util.List;
import javax.persistence.*;

@Entity
@Table(name = "schedule")
public class Schedule implements Serializable {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    private LocalDate planningDate;

    private Integer relationId;

    @OneToMany(fetch = FetchType.LAZY, mappedBy = "scheduleid")
    private List<Activity> activities;

    private boolean holiday;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public LocalDate getPlanningDate() {
        return planningDate;
    }

    public void setPlanningDate(LocalDate planningDate) {
        this.planningDate = planningDate;
    }

    public Integer getRelationId() {
        return relationId;
    }

    public void setRelationId(Integer relationId) {
        this.relationId = relationId;
    }

    public List<Activity> getActivities() {
        return activities;
    }

    public void setActivities(List<Activity> activities) {
        this.activities = activities;
    }

    public boolean getHoliday() {
        return this.holiday;
    }

    public void setHoliday(boolean holiday) {
        this.holiday = holiday;
    }
}
